src/Workspaces/Core/Portable/Workspace/Solution/ProjectInfo.cs

```diff
@@ -238,7 +240,7 @@ public sealed class ProjectInfo
                 id, version, name, assemblyName, language,
                 filePath, outputFilePath, outputRefFilePath: null, defaultNamespace: null, compilationOptions, parseOptions,
                 documents, projectReferences, metadataReferences, analyzerReferences, additionalDocuments, analyzerConfigDocuments: null,
-                isSubmission, hostObjectType, hasAllInformation: true, runAnalyzers: true);
+                isSubmission, hostObjectType, hasAllInformation: true, runAnalyzers: true, telemetryId: default);
```

Comment:
Should that other Create just have default parameters to avoid having to write this twice?

Reply:
I thought it likely didn't have default parameters to force you to be explicit about what your were getting from these legacy constructors.